Pointyware/XYZ
# Feature/18 request a ride

Resolves #18
* Create RequestRideUiTest
* Implement composables to pass test
* Steps:
  * Idle
  * Search For Location; Select Suggestion
  * Wait for Route to load
  * Confirm Route
